scania/corporate-ui-site
# Improvement - Adding table with properties and attribute for each web-component

**Describe pull-request**  
_Describe what the pull-request is about_
- Adding table with properties and attribute for each web-component
- Change detection for the props
- Small indetation fixes
- Added a instance of the component inside code-example

**Needed** 
Add a new content type field for each section(JSON Field)

**Solving issue**  
_Add which issue this pull-request solves by adding # plus the number of the issue (for example #123)_
Fixes: #87 

**How to test**  
_Add description how to test if possible_

 You need to do some changes in CMS
1.  Add a new content type for the each content. A json field
2. Go to a component and add the same line which i shown in the picture for one of the web-components, for example c-cookie. The json needs to match what is shown in the picture
![image](https://user-images.githubusercontent.com/35451568/93475149-9692c380-f8f8-11ea-83c8-6fdcd29c8443.png)
![image](https://user-images.githubusercontent.com/35451568/93475043-7400aa80-f8f8-11ea-9a43-0574919451d5.png)
3. When everything is saved you can now fetch the data in the site
3. Go the the site and look at the web-component, there should be a table showing the props
4. Go to a regular component and see that there is no table
